TidepoolOnboarding/Views/Sections/YourDevicesViews.swift

```diff
@@ -56,8 +57,15 @@ fileprivate struct YourDevicesNotificationsView: View {
                 LocalizedString("Enable Critical Alerts in your iPhone or iPod Touch Settings", comment: "Onboarding, Your Devices section, Notifications view, list 2, item 1")
             )
             .startingAt(2)
-            .padding(.vertical)
+            .padding(.top)
             Paragraph(LocalizedString("Critical Alerts may be configured to alert you to higher risk situations while using Tidepool Loop, such as urgent low glucose, insulin pump occlusions, or other serious system errors.", comment: "Onboarding, Your Devices section, Notifications view, paragraph 3"))
+            .padding(.vertical)
+            Callout(title: LocalizedString("Notifications and Critical Alert permissions must be allowed to continue using the app", comment: "Onboarding, Your Devices section, Notifications view, callout title"),
+                    warningIconColor: .red) {
```

Comment:
[nit] Seems silly to me to have two different warning colors.

Reply:
I'm just following the [design spec](https://www.figma.com/file/JFIqEtYArG0oMWJCgLbQL5/LOOP-3644-3645?node-id=201%3A2720). 🤷‍♂️

Reply:
:+1: